e2e/jenkins/crw-ci/k8s/pr-check/Jenkinsfile

```diff
@@ -0,0 +1,229 @@
+#!groovy
+
+pipeline {
+    agent { label 'codenvy-slave9' }
+
+    options {
+        timestamps()
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '', daysToKeepStr: '60', numToKeepStr: '100'))
+    }
+
+    // triggers { cron("@midnight") }
+
+    environment {
+        PATH = "/tmp:/qa/tools/opt/apache-maven-3.5.4/bin:$PATH"
+        JENKINS_BUILD = "true"
+
+        DEVFILE_URL = "${WORKSPACE}/e2e/files/happy-path/happy-path-workspace.yaml"
```

Comment:
Missing `che` directory? `${WORKSPACE}/che/e2e/files/happy-path/happy-path-workspace.yaml`?

Reply:
Good catch!
There is `che` project content in the root of workspace directory on `codenvy-slave9`, which had been checked out at one of the previous build of the job.